[Releases3DS] Add support for Switch releases

ORelio opened the pull request:
_This pull request contains changes separated from #1507._

Request nswdb.com instead of 3dsdb.com for Switch releases.
The two websites are identical, only the URL changes.

ORelio committed:
[Releases3DS] Add support for Switch releases

Request nswdb.com instead of 3dsdb.com for Switch releases

em92 commented:
https://feed.eugenemolotov.ru/pr1694/?action=display&bridge=Releases3DS&console=switch&format=Html

1.
```
Notice: Trying to get property 'href' of non-object in /var/www/rss-bridge/pr1694/bridges/Releases3DSBridge.php on line 73

Notice: Trying to get property 'src' of non-object in /var/www/rss-bridge/pr1694/bridges/Releases3DSBridge.php on line 71
```

2. I chose "console: switch" but feed title links to 3ds. AFAIK 3ds and switch are different consoles from one manufacturer Nintendo.

Have you tried to make a separate bridge for nswdb just inheriting from 3dsdb bridge?

ORelio commented:
Thanks for reporting the notices, I'll have a look.
Making a separate bridge with inheritance could work, I'll attempt this 👍

ORelio committed:
[Releases3DS] Fix PHP notices related to IGN

ORelio committed:
[ReleasesSwitch] Switch scene releases

Separate bridge from Releases3DS that just has a different URL.
Inherits from Releases3DS so both bridges need to be present.

ORelio commented:
Notices fixed and separate bridge done. Of course the separate bridge requires Releases3DS to be present.

em92 commented:
gj @ORelio !
